src/alpine/3.10/helix/arm64v8/Dockerfile

```diff
@@ -46,21 +46,9 @@ RUN apk update && \
 # Install Helix Dependencies
 
 RUN ln -sf /usr/bin/python3 /usr/bin/python && \
```

Comment:
This symlink might be problematic: see https://www.python.org/dev/peps/pep-0394/.
Since Python2 and Python3 should be treated as different languages (although very close ones), choosing to link python3 to python (as opposed to python2) might break some of the software (e.g., that gets installed from the packaging system). This might not be a concern for us when we run only Helix, but in general, this might lead to tricky issues. Is there a reason we don't call python3 explicitly to invoke Helix scripts?

Reply:
The Helix python entry point is always the environment variable $HELIX_PYTHONPATH. This was also not introduced in this PR and is not really relevant to this conversation.  Furthermore, Python 2 is end of life in 13 days so it really doesn't matter if I break it IMO.

Since the user selects the image they use which sets this value, and if they want to use Helix's python functionality must use Python 3, it's simply never been a problem.  Some users DO still use Python 2 docker image; they use $HELIX_PYTHONPATH regardless. If it becomes a problem in the next 13 days, we can look into changing it, otherwise the version is obsolete.